_explore/scripts/get_dependency_info.py

```diff
@@ -0,0 +1,62 @@
+from scraper.github import queryManager as qm
+from os import environ as env
+
+ghDataDir = env.get("GITHUB_DATA", "../github-data")
+datfilepath = "%s/dependencyInfo.json" % ghDataDir
+queryPath = "../queries/dependency-Info.gql"
+
+# Read repo info data file (to use as repo list)
+inputLists = qm.DataManager("%s/labRepos_Dependencies.json" % ghDataDir, True)
+# Populate repo list
+repolist = []
+print("Getting internal repos ...")
+for repoName in inputLists.data["data"]:
+    for node in inputLists.data["data"][repoName]["dependencyGraphManifests"]["nodes"]:
+        for repo in node["dependencies"]["nodes"]:
+            if (
+                repo["repository"] is not None
+                and repo["repository"]["nameWithOwner"] is not None
+            ):
+                repolist.append(repo["repository"]["nameWithOwner"])
+repolist = list(dict.fromkeys(repolist))
+repolist = sorted(repolist)
+print("Repo list complete. Found %d repos." % (len(repolist)))
+
+# Initialize data collector
+dataCollector = qm.DataManager(datfilepath, False)
+dataCollector.data = {"data": {}}
+
+# Initialize query manager
+queryMan = qm.GitHubQueryManager()
+
+# Iterate through internal repos
+print("Gathering data across multiple paginated queries...")
+for repo in repolist:
+    print("\n'%s'" % (repo))
+
+    r = repo.split("/")
+    try:
+        outObj = queryMan.queryGitHubFromFile(
+            queryPath,
+            {"ownName": r[0], "repoName": r[1],},
+            paginate=False,
+            cursorVar="pgCursor",
+            keysToList=["data", "languages", "owner"],
+            headers={"Accept": "application/vnd.github.hawkgirl-preview+json"},
+        )
```

Comment:
```suggestion
        outObj = queryMan.queryGitHubFromFile(
            queryPath,
            {"ownName": r[0], "repoName": r[1],},
            headers={"Accept": "application/vnd.github.hawkgirl-preview+json"},
        )
```
This query is not paginated, so these 3 variables are unnecessary. ([See scraper documentation.](https://github.com/LLNL/scraper/blob/8b76a79b1f17cf849a820a7ac83ff36dee4b16cb/scraper/github/queryManager.py#L208-L218))